content/docs/conditional-rendering.md

```diff
@@ -116,13 +116,13 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Coba di Codepen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Saat mendeklarasikan sebuah variabel dan  menggunakan statement `if` merupakan cara yang baik merender komponen secara kondisional, terkadang Anda mungkin ingin memakai sintaksis pendek. Ada beberapa cara _iniline_ bersyarat pada JSX, dijelaskan dibawah.
```

Comment:
```suggestion
Saat mendeklarasikan sebuah variabel dan menggunakan statement `if` merupakan cara yang baik merender komponen secara kondisional, terkadang Anda mungkin ingin memakai sintaksis pendek. Ada beberapa cara _inline_ bersyarat pada JSX, dijelaskan dibawah.
```

Remove extra space and typo on `inline`. Also, `merender` and `sintaksis` should be discussed further.